ui.go

```diff
@@ -519,6 +542,26 @@ func (win *win) printDir(ui *ui, dir *dir, context *dirContext, dirStyle *dirSty
 	}
 }
 
+func getGroupWidth(dir *dir, beg int, end int) int {
```

Comment:
You mixed up the function names:

```suggestion
func getUserWidth(dir *dir, beg int, end int) int {
```